assets/cards-skinny/credit.md

```diff
@@ -0,0 +1 @@
+The cards in this deck were created by an anonymous person who gave the work to virtualtabletop.io.  Virtualtabletop.io releases all of this to the Public Domain under CC0.
```

Comment:
Hum isn't there a source? Shouldn't we give credits to the username?

Also was it given directly to VTT or was it made public and we took it?
The "given to VTT" and "we are releasing it as CC0" makes me feel there is something wrong.

Reply:
That's unrelated to the PR though. The same text was in the attribution of the variant before.

Reply:
> Hum isn't there a source? Shouldn't we give credits to the username?
> 
> Also was it given directly to VTT or was it made public and we took it? The "given to VTT" and "we are releasing it as CC0" makes me feel there is something wrong.

The person who made this deck did not want attribution by name.  They made it, gave it to VTT to do with whatever.  I have the message traffic somewhere in Discord.